src/DurableTask/Grpc/ProtoUtils.cs

```diff
@@ -176,6 +175,29 @@ static Timestamp ConvertTimestamp(DateTime dateTime)
         return Timestamp.FromDateTime(dateTime);
     }
 
+    internal static Timestamp? ToTimestamp(this DateTime? dateTime)
+        => dateTime.HasValue ? dateTime.Value.ToTimestamp() : null;
+
+    internal static Timestamp ToTimestamp(this DateTimeOffset dateTime) => Timestamp.FromDateTimeOffset(dateTime);
+
+    internal static Timestamp? ToTimestamp(this DateTimeOffset? dateTime)
+        => dateTime.HasValue ? dateTime.Value.ToTimestamp() : null;
+
+#pragma warning disable 0618
```

Comment:
Which warning is this that we're disabling? Can you add a comment to help make that more clear?